src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -130,19 +162,84 @@ const Backdrop = styled('div')<Openable>`
   }};
 `
 
+const renderUpsellCard = (
+  quoteBundleVariants: QuoteBundleVariant[],
+  selectedQuoteBundleVariant: QuoteBundleVariant,
+  onAcceptDeal: (quoteIds: string[]) => void,
+  textKeys: Record<string, any>,
+) => {
+  const isSwedishAccidentAdded = selectedQuoteBundleVariant.bundle.quotes.some(
+    ({ quoteDetails }) => isSwedishAccident(quoteDetails),
+  )
+  if (isSwedishAccidentAdded) {
+    return null
+  }
```

Comment:
I would prefer to hide this market-specific information under a feature flag. We should consider all features as global -- but possibly only enabled in a few markets at the moment.